setup.py

```diff
@@ -0,0 +1,29 @@
+import setuptools
+import versioneer
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+
+setuptools.setup(
+    name="versioned-hdf5",
+    version=versioneer.get_version(),
+    cmdclass=versioneer.get_cmdclass(),
+    author="Quansight",
+    description="Versioned HDF5",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Quansight/versioned-hdf5",
+    packages=['versioned_hdf5', 'versioned_hdf5.tests'],
+    license="MIT",
+    install_requires=[
+        "h5py",
+        "numpy",
```

Comment:
Isn't numpy an implicit dependency? We don't link against it.

Reply:
We import numpy in several of the files.

Reply:
Sure, but h5py requires numpy

Reply:
I generally like to list explicitly used dependencies, even if they are already implied by other dependencies.